### PR TITLE
Update caching-your-github-credentials-in-git.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
### What's being changed:

I fixed a typo and an improperly capitalized letter.

### CODE REVIEW

File: content/get-started/getting-started-with-git/caching-your-github-credentials-in-git.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Once you've authenticated successfully, your credentials are stored in the Windo
 
 {% linux %}
 
-For Linus, Install Git and GCM Core, then configure Git to use GCM Core.
+For Linux, install Git and GCM Core, then configure Git to use GCM Core.
```
Comment (Muna58009924) [inline content/get-started/getting-started-with-git/caching-your-github-credentials-in-git.md:68]: Thanks

**Review by Muna58009924 — APPROVED** [review]

**Review by Muna58009924 — APPROVED** [review]

File: content/get-started/getting-started-with-git/caching-your-github-credentials-in-git.md (hunk 1 above)
Comment (Eldrick99) [reply content/get-started/getting-started-with-git/caching-your-github-credentials-in-git.md:68]: https://github.com/github/docs/pull/8942

**Review by ramyaparimi — APPROVED** [review]
@oddlittlebird 
Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Comment by amirnateghi1980** [comment]
> @oddlittlebird
> Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Pull request merged**